packages/amplify_flutter/android/build.gradle

```diff
@@ -58,13 +58,13 @@ android {
 
 dependencies {
     api amplifyCore
-    implementation 'com.amplifyframework:core:1.6.10'
+    implementation 'com.amplifyframework:core:1.16.13'
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     testImplementation 'junit:junit:4.13'
     testImplementation 'org.mockito:mockito-core:3.1.0'
     testImplementation 'org.mockito:mockito-inline:3.1.0'
     testImplementation 'androidx.test:core:1.3.0'
     testImplementation 'org.robolectric:robolectric:4.3.1'
     testImplementation 'com.google.code.gson:gson:2.8.6'
-    testImplementation 'com.amplifyframework:aws-auth-cognito:1.6.10'
+    testImplementation 'com.amplifyframework:aws-auth-cognito:1.16.13'
```

Comment:
Is this needed?

Reply:
I don't think this is needed - it's a test dependency, so we can remove it later without problems.